app/main.py

```diff
@@ -1,2 +1,72 @@
+class Node:
+    def __init__(self, key: str, value: int) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+        self.next = None
+
+
 class Dictionary:
-    pass
+    def __init__(self, initial_capacity: int = 8) -> None:
+        self.capacity = initial_capacity
+        self.size = 0
+        self.buckets = [None] * self.capacity
+        self.load_factor = 0.75
+
+    def __setitem__(self, key: str, value: int) -> None:
+        index = self._get_index(key)
+        node = self.buckets[index]
+
+        if node is None:
+            self.buckets[index] = Node(key, value)
+            self.size += 1
+        else:
+            while node:
+                if node.key == key:
+                    node.value = value
+                    return
+                if node.next is None:
+                    node.next = Node(key, value)
+                    self.size += 1
+                    break
+                node = node.next
+
+        if self.size / self.capacity > self.load_factor:
+            self._resize()
+
+    def __getitem__(self, key: str) -> None:
```

Comment:
The return type of the `__getitem__` method should not be `None`. It should return an `int` since it returns the value associated with the key.